tests/test_infer/test_inference_engine.py

```diff
@@ -0,0 +1,31 @@
+import transformers
+from transformers import AutoTokenizer
+
+from colossalai.inference.config import InferenceConfig
+from colossalai.inference.core.engine import InferenceEngine
+
+
+def test_inference_engine():
+    model = transformers.LlamaForCausalLM(
+        transformers.LlamaConfig(
+            vocab_size=20000, hidden_size=512, intermediate_size=1536, num_attention_heads=4, num_hidden_layers=4
+        )
+    )
+    tokenizer = AutoTokenizer.from_pretrained("hf-internal-testing/llama-tokenizer")
+    inference_config = InferenceConfig(model, tokenizer)
+    inference_engine = InferenceEngine(inference_config, verbose=True)
```

Comment:
`model` and `tokenizer` should be passed to `InferenceEngine` instead of `InferenceConfig`.